WooCommerce/Classes/ViewRelated/Products/Variations/Add Attributes/AddAttributeOptionsViewModel.swift

```diff
@@ -18,14 +18,21 @@ final class AddAttributeOptionsViewModel {
 
     /// Defines the necessary state to produce the ViewModel's outputs.
     ///
-    private struct State {
+    fileprivate struct State {
+        /// Bundles the attributes options that are supported
+        ///
+        enum Option {
+            case local(name: String)
+            case global(option: ProductAttributeTerm)
+        }
+
         /// Stores the options to be offered
         ///
-        var optionsOffered: [String] = []
+        var optionsOffered: [Option] = []
 
         /// Stores options previously added
         ///
-        var optionsAdded: [ProductAttributeTerm] = []
+        var optionsAdded: [Option] = []
```

Comment:
nit: I first thought `optionsAdded` is for the options added by the user on this Add Options screen, would it be less confusing if this variable is called like `options`/`allOptions`/`optionsAvailable`? The "Add" action in "AddAttributeOptions*" actually means adding an existing option to be offered, if I understand correctly?

Reply:
I'll rename those properties in order for it to be more clear! Thanks for flagging that up! :-)